Candidate: CVE-2021-41496
PublicDate: 2021-12-17 20:15:00 UTC
References:
 https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-41496
 https://github.com/numpy/numpy/issues/19000
 https://github.com/numpy/numpy/pull/20630
 https://github.com/numpy/numpy/commit/271010f1037150e95017f803f4214b8861e528f2
Description:
 ** DISPUTED ** Buffer overflow in the array_from_pyobj function of
 fortranobject.c in NumPy < 1.19, which allows attackers to conduct a Denial
 of Service attacks by carefully constructing an array with negative values.
 NOTE: The vendor does not agree this is a vulnerability; the negative
 dimensions can only be created by an already privileged user (or
 internally).
Ubuntu-Description:
Notes:
Mitigation:
Bugs:
Priority: medium
Discovered-by:
Assigned-to: 0xnishit
CVSS:
 nvd: CVSS:3.1/AV:L/AC:L/PR:L/UI:N/S:U/C:N/I:N/A:H [5.5 MEDIUM]


Patches_numpy:
 upstream: https://github.com/numpy/numpy/commit/271010f1037150e95017f803f4214b8861e528f2
upstream_numpy: released (1.22.0, 1.23.0)
trusty_numpy: ignored (out of standard support)
xenial_numpy: ignored (out of standard support)
focal_numpy: needs-triage
hirsute_numpy: ignored (reached end-of-life)
impish_numpy: ignored (reached end-of-life)
jammy_numpy: needs-triage
kinetic_numpy: needs-triage
devel_numpy: needs-triage
